predictions/app/src/main/java/com/google/firebase/example/predictions/ConditionalAdsActivity.java

```diff
@@ -16,8 +16,6 @@
 
 public class ConditionalAdsActivity extends AppCompatActivity {
 
-    private static final long CACHE_EXPIRATION = 60 * 1000;
```

Comment:
Note to reviewer: This constant was being passed as argument to the (now deprecated) `fetch()` function. The new `fetchAndActivate() `function doesn't require it, since fetch interval should now be specified while configuring Remote Config.